spec/bolt_spec/run_spec.rb

```diff
@@ -81,4 +83,39 @@
       expect(result['value']['kind']).to eq('bolt/run-failure')
     end
   end
+
+  describe 'apply_manifest' do
```

Comment:
These tests will require a different config (there is a `facts` module on the modulepath and it tries to install puppet agent with `bolt` user). Will need to update.